### PR TITLE
add(epp) : applications-nginx-plus-restapi FR

### PR DESCRIPTION
**Pull request by psamecentreon**

### CODE REVIEW

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+<!--Connections-->
+
+| Metric name                 | Description                    |
+| :-------------------------- | :----------------------------- |
+| connections.active.count    | Number of active connections   |
+| connections.idle.count      | Number of idle connections     |
+| connections.accepted.count  | Number of accepted connections |
+| connections.dropped.count   | Number of dropped connections  |
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:36]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+| http.$name.zone.responses.total.count                   | Number total of http-zone responses                      |
+| http.$name.zone.responses.[1xx,2xx,3xx,4xx,5xx].count   | Number 1xx,2xx,3xx,4xx,5xx of http-zone responses        |
+
+<!--Ssl-->
+
+| Metric name                      | Description                     |
+| :------------------------------- | :------------------------------ |
+| ssl.handshakes.succeeded.count   | Number of handshakes succeeded  |
+| ssl.handshakes.failed.count      | Number of handshakes failed     |
+| ssl.sessions.reuses.count        | Number of sessions reuses       |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:55]: SSL handshakes and SSL sessions ;-)

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:56]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+### Privilèges API
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:62]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Un compte de service est requis pour interroger l'API Nginx Plus. Celui-ci doit avoir suffisamment de privilèges en lecture dans l'environnement.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:64]: Link to the official documentation ?

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-applications-nginx-plus-restapi.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Nginx Plus* depuis la page "Configuration > Plugin packs > Manager"
+
+
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:96]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:101]:
```suggestion
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par environnement Nginx Plus
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle *App-Nginx-Plus-Restapi-custom*. Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory | Name        | Description                                      |
+| :-------- | :---------- | :----------------------------------------------- |
+| X         | APIPORT     | Port used (Default: 443)                         |
+| X         | APIPROTO    | Specify https if needed (Default: 'https')       |
+| X         | APIUSERNAME | Nginx basic username                             |
+| X         | APIPASSWORD | Nginx basic password.                            |
+| X         | APIPATH     | Specify api path (Default: '/api/6')             |
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:112]: No extra options ?

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+
+## FAQ
+
+### Comment tester un contrôle en ligne de commandes et que signifient les options principales ?
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:117]:
```suggestion
### Comment puis-je tester le Plugin et que signifient les options des commandes ?
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commandes depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:119]:
```suggestion
Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commande depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+	--filter-counters='mycountersfilter' \
+	--warning-active='60' \
+	--critical-active='80'
+	--warning-idle='8' \
+	--critical-idle='10' \
+    --warning-accepted='50' 
+	--critical-accepted='65' \
+	--warning-dropped='3' \
+    --critical-dropped='5' \
+	--verbose
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:139]: Align the options

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+	
+
+OK: Active : 5, Idle : 0, Accepted : 5, Dropped : 0|
+'connections.active.count'=5;;60;80; 'connections.idle.count'=1;;8;10; 'connections.accepted.count'=5;;50;65; 'connections.dropped.count'=0;;3;5;
+```
+
+La commande ci-dessus contrôle les connexions de Nginx Plus (```--mode=connections```).
+Cette commande déclenchera une alarme WARNING si les connexions activent dépasse 60 (```--warning-active='60'```) 
+et une alarme CRITICAL s'il dépasse 80 (```--critical-active='80'```).
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:148]: Enhance the explanation about the options and the triggers (or remove some trigger in your example).

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl --plugin=apps::nginx::nginxplus::restapi::plugin --mode=connections --help```
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:153]:
```suggestion
```
/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl --plugin=apps::nginx::nginxplus::restapi::plugin --mode=connections --help
```
```

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+
+
+### Comment puis-je supprimer les perfdatas *count* dans le cas où je ne souhaite vérifier qu'une seule application ?
+
+Le Plugin permet de filtrer sur un ou plusieurs éléments mais permet également de récupérer l'ensemble des éléments si aucun filtre n'est spécifié.
+De ce fait, des perfdatas "globales" sur les statistiques des objets sont ajoutées par défaut. Il est possible de supprimer ces données de performance en appliquant le filtre suivant: ```--filter-perfdata='^$'```.
```
Comment (pkriko) [inline fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md]: Add documentation about the most commons error when using restapi plugin:
https://docs.centreon.com/current/fr/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.html#jobtiens-le-message-derreur-suivant

**Review by pkriko — CHANGES_REQUESTED** [review]

File: fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md (hunk 5 above)
Comment (psamecentreon) [reply fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md:60]: Done

**Review by pkriko — APPROVED** [review]

**Pull request merged**